roles/destroy_vms/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 - name: Create inventory host to vm name mapper
   set_fact:
-    cluster_vm_names: "{{ (cluster_vm_names | default([])) + [item.startswith(cluster_name) | ternary(item, (cluster_name + '_' + item))] }}"
```

Comment:
I think the `item.startswith(cluster_name)` should be `item.startswith(vm_mode_prefix)`. I checked `'a random string'.startswith('') == True`

Reply:
Thanks for doing the test for me..  I did think about doing exactly that..  but I didn't think it would match for ''

I'll update the PR